### ISSUE TITLE
Wrong today's date.

### ISSUE THREAD

**Issue by aavrug** [donatj/SimpleCalendar#6]
I am displaying this month's calendar but 01-09-2016 is selected by default instead of today's date.
$calendar = new \donatj\SimpleCalendar('September '.$currentYear);
$calendars['month'] = $monthName;
$calendars['calendar'] = $calendar;
$calendars['calendar']->show();

**Comment by donatj** [comment]
The following outputs correctly for me. How about you?

```php
$calendar = new \donatj\SimpleCalendar();
$calendar->setDate('September 2016');
$calendar->show(true);
```

There's very little to the setDate parsing. The method `setDate` simply calls `getdate(strtotime($date_string))`

If you execute `var_export(getdate(strtotime('September 2016')));` do you get the following?

```
array (
  'seconds' => 0,
  'minutes' => 0,
  'hours' => 0,
  'mday' => 1,
  'wday' => 4,
  'mon' => 9,
  'year' => 2016,
  'yday' => 244,
  'weekday' => 'Thursday',
  'month' => 'September',
  0 => 1472680800,
)
```

**Comment by aavrug** [comment]
Yea I am also getting same output like your's but you have also default date selected is 1. Does this means In this plugin I cannot show today's date selected by default?

**Comment by donatj** [comment]
Oops, didn't mean to close that ;)

**Comment by donatj** [comment]
Fixed in #12
